src/trackingModules/CanvasTracking.js

```diff
@@ -0,0 +1,53 @@
+export const CANVAS_EVENTS = {
+  '.djs-context-pad .entry': {
```

Comment:
What about drag start? Asking because distinguishing it allows us to learn more about usage patterns.

Reply:
I mean the fact that we allow drag start but also click from any of the tools on the canvas:

![capture RmzNgr_optimized](https://user-images.githubusercontent.com/58601/184348130-cb87dcb6-1f07-437e-bb24-9b6a5808cf1c.gif)

It is important to understand how people use our tools (via dragstart or click) and most of all capture all interactions.